### PR TITLE
Add Ente

### PR DESCRIPTION
**Pull request by fleish**
This pull request adds ente.io, which offers end-to-end encrypted file/photo sharing as well as an TOTP Authenticator app

### CODE REVIEW

File: entries/e/ente.io.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "Ente": {
+    "domain": "ente.io",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://ente.io/faq/security-and-privacy/two-factor-support/",
```
Comment (hkamran80) [inline entries/e/ente.io.json]: This URL doesn't explain how to set up MFA.

```suggestion
```

File: entries/e/ente.io.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "categories": [
+      "backup",
+      "cloud",
+      "security"
```
Comment (hkamran80) [inline entries/e/ente.io.json]: The `cloud` category is for serverless cloud computing and `security` is for "online security and anti-malware services". For more information, check out the [categories list](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#categories).

```suggestion
      "backup"
```

File: img/e/ente.io.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...dAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="17.939"/><feGaussianBlur stdDeviation="8.969"/><feComposite in2="hardAlpha" k2="-1" k3="1" operator="arithmetic"/><feColorMatrix values="0 0 0 0 1 0 0 0 0 1 0 0 0 0 1 0 0 0 0.25 0"/><feBlend in2="shape" result="effect1_innerShadow_26133_116870"/><feColorMatrix in="SourceAlpha" result="hardAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="-8.969"/><feGaussianBlur stdDeviation="4.485"/><feComposite in2="hardAlpha" k2="-1" k3="1" operator="arithmetic"/><feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.25 0"/><feBlend in2="effect1_innerShadow_26133_116870" result="effect2_innerShadow_26133_116870"/></filter><linearGradient id="b" x1="96.5" x2="985" y1="71" y2="977.5" gradientUnits="userSpaceOnUse"><stop stop-color="#C373F2" style="stop-color:#c373f2;stop-color:color(display-p3 .7647 .451 .949);stop-opacity:1"/><stop offset="1" stop-color="#9610D6" style="stop-color:#9610d6;stop-color:color(display-p3 .5882 .0627 .8392);stop-opacity:1"/></linearGradient><clipPath id="a"><rect width="1024" height="1024" fill="#fff" rx="234" style="fill:#fff;fill-opacity:1"/></clipPath></defs></svg>
```
Comment (hkamran80) [inline img/e/ente.io.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1024 1024"><rect width="1024" height="1024" fill="#3eb54a" ry="182.3"/><path fill="#fff" d="M791 306.8a280.2 280.2 0 0 0-111-101.5 332 332 0 0 0-153.7-35.4c-60.7 0-114.7 12.8-162.2 38a283.9 283.9 0 0 0-112 105.2 291 291 0 0 0-40.7 153.7c0 56.4 13.9 107 41.7 151.6a278 278 0 0 0 76.7 81.1c.8.7 1.8 1.2 2.6 1.8a551.9 551.9 0 0 0 17 11.1c6.8 4.2 13.8 8 20.9 11.6 50.4 25.8 109.7 38.5 178 38.5a410 410 0 0 0 119-15.9c6.3-2 12.4-4 18.5-6.2l3.6-1.4.5-.1 7.5-3a261.2 261.2 0 0 0 92.9-65.1l-104.5-111a222 222 0 0 1-59.2 37.6 192 192 0 0 1-72.9 12 173 173 0 0 1-78.1-16.3 117 117 0 0 1-62-75.6h415.9l2.1-25.3c.7-9 1-17.1 1-24.2 0-63.5-13.9-117.2-41.7-161.2ZM410.3 411.9c2.3-13.7 6.3-26.1 11.8-37.5a103.8 103.8 0 0 1 41.8-45 126.8 126.8 0 0 1 64.4-15.8c24 0 44.7 5.3 62.4 15.8a111.9 111.9 0 0 1 41.6 45c6.1 11.4 10.1 24 12.4 37.5H410.4Z"/></svg>
```

File: entries/e/ente.io.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    ]
```
Comment (hkamran80) [inline entries/e/ente.io.json:16]: > Users will not knowingly directly or indirectly export, re-export, transfer, make available or release (collectively, "Export") our Services, Services to any destination, person, entity or end-use prohibited or restricted under the applicable law without prior US government authorization to the extent required by the applicable export control regulations, including without limitation, to any parties listed on any of the denied parties lists or specially designated nationals lists to the extent required by the applicable regulations.
> *— Terms of Service, Article 17*

```suggestion
    ],
    "regions": [
      "-cu",
      "-ir",
      "-kp",
      "-ru",
      "-sy"
    ]
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/e/ente.io.json (hunk 1 above)
Comment (fleish) [reply entries/e/ente.io.json]: It doesn't go step by step, but it does provide a pointer on where to go to set it up. I've already set it up on my client, but you can see it here on the desktop client

Screenshot 2024-10-24 at 9 12 58 AM

Screenshot 2024-10-24 at 9 13 07 AM

File: entries/e/ente.io.json (hunk 2 above)
Comment (fleish) [reply entries/e/ente.io.json]: I agree cloud is a mis-tag, but backup also mentions "cross-device file synchronization" which I believe applies to the Ente app

Comment (hkamran80) [reply entries/e/ente.io.json]: Backup is still in the suggestion. I removed `cloud `and `security`.

File: entries/e/ente.io.json (hunk 1 above)
Comment (hkamran80) [reply entries/e/ente.io.json]: If it provided step-by-step instructions like [Calendly](https://help.calendly.com/hc/articles/26682031653399), for example, then the `documentation` URL could be added.

Plus, a good assumption for where MFA configuration would be is in settings.

File: entries/e/ente.io.json (hunk 2 above)
Comment (fleish) [reply entries/e/ente.io.json]: My bad, I misread the diff

File: entries/e/ente.io.json (hunk 1 above)
Comment (fleish) [reply entries/e/ente.io.json]: I'll refer the developers to this and if they want to make adjustments they can

**Comment by fleish** [comment]
@fletcherdodd @hkamran80 not sure why the categories test is failing now, runner claims it cannot find the categories.json file so tagging this for review in case someone needs to manually intervene

**Comment by hkamran80** [comment]
That would be because we moved the file. We can still review and merge it though.

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**